### ISSUE TITLE
When a promscrape config contains a target that contains the scheme, vmagent will drop the target without an error message

### ISSUE THREAD

**Issue by denisgolius** [VictoriaMetrics/VictoriaMetrics#3208]
**Describe the bug**
 when a promscrape config contains a target that contains the scheme (http/https), vmagent will drop the target without an error message

**To Reproduce**
add a http/https scheme to scrape target, see an examples below:

**prometheus.yml**:
```yaml
global:
  scrape_interval: 10s

scrape_configs:
  - job_name: 'vmagent'
    static_configs:
      - targets: ['http://vmagent:8429']
  - job_name: 'vmalert'
    static_configs:
      - targets: ['vmalert:8880']
  - job_name: 'victoriametrics'
    static_configs:
      - targets: ['victoriametrics:8428']

```

**Expected behavior**
vmagent will not start or print out an error to logs/stdout as it works in prometheus 

**Screenshots**
![image](https://user-images.githubusercontent.com/5650611/194561103-2d2ed955-e4dc-4eb1-b016-aecfd02d86af.png)

**Comment by valyala** [comment]
This is a kind of a feature-bug :) Prometheus doesn't allow specifying scheme and metrics path in `targets` list and in the  `__address__` label - see the `__address__ contains "/"` block at the first diagram in [this article](https://www.robustperception.io/life-of-a-label/).

I think this is more a bug than a feature, since it feels natural specifying full scrape urls in `targets` list or in `__address__` label during [relabeling](https://docs.victoriametrics.com/vmagent.html#relabeling). So this "bug" has been fixed in the commit f9df0cae16438beeb8245a95bc11406a0bd44842 . This commit will be included in the next release. In the meantime it is possible to build `vmagent` from this commit according to [these instructions](https://docs.victoriametrics.com/vmagent.html#how-to-build-from-sources) and verify whether it accepts full urls in `targets` list as expected.

**Comment by valyala** [comment]
`vmagent` accepts full urls at `__address__` label and at `targets` list of [static_configs](https://docs.victoriametrics.com/sd_configs.html#static_configs) starting from [v1.83.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases). Closing this issue as fixed.
